src/include/daos/common.h

```diff
@@ -517,4 +517,16 @@ daos_unparse_ctype(daos_cont_layout_t ctype, char *string)
 	}
 }
 
+static inline int daos_gettime_corase(uint64_t *time)
+{
+	struct timespec	now;
+	int		rc;
+
+	rc = clock_gettime(CLOCK_MONOTONIC_COARSE, &now);
+	if (rc == 0)
+		*time = now.tv_sec + (now.tv_nsec >> 30);
```

Comment:
Isn't the (now.tv_nsec >> 30) is always zero? I was thinking the 'tv_nsec' in timespec is always less than 1 second, is that true?

Reply:
Then I am confused by the implementation:

static inline double
d_time2s(struct timespec t)
{
        return (double) t.tv_sec + (double) t.tv_nsec / 1e9;
}